gitleaks-report.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Is the output file supposed to be committed?

Reply:
NOPE! thanks